norfair/metrics.py

```diff
@@ -210,14 +210,14 @@ def compute_metrics(
         if metrics is None:
             metrics = list(mm.metrics.motchallenge_metrics)
 
-        self.summary = eval_motChallenge(
+        self.summary, self.summary_pd = eval_motChallenge(
             matrixes_predictions=self.matrixes_predictions,
             paths=self.paths,
             metrics=metrics,
             generate_overall=generate_overall,
         )
 
-        return self.summary
+        return self.summary_pd
```

Comment:
You mentioned you needed this change for the demo, but I am not seeing it getting used in the demo? Where do we see the effect of this change? Also what is the difference between `summary` and `summary_pd`?

Reply:
The main difference is that summary is a string, so you can write your evaluation results on a text file or print them on terminal. Now, summary_pd is a pandas dataframe, so it lets you access to each individual value in the corresponding table. I did not use summary_pd in this particular demo, but I thought it could be handy to have it as a feature in the Accumulator class, in case someone using your library needed it

Reply:
Gotcha. Is this used in other parts of Norfair? Will it break the other demos? I see that the old summary is no longer returned by the function compute_metrics, so it is possible it will not be easy to access.

Reply:
`summary_pd` is not used in any other part of norfair, having or not having `summary_pd` will not break any demo. The old summary (the string one) is not returned in `compute_metrics`, but is used in the functions `print_metrics` and `save_metrics`. Anyway, you can access both summaries really easy, as they are both saved in your `Accumulator` type variable as `self.summary` and `self.summary_pd`.